microph/fortran/main.F

```diff
@@ -64,7 +64,7 @@ program test
 
     do tile = 0, 0 ! num_tiles - 1
     
-    !$ser init directory='./data' prefix='Serialized' prefix_ref='Generator' mpi_rank=tile
+    !$ser init directory='../data' prefix='Serialized' prefix_ref='Generator' mpi_rank=tile
```

Comment:
Great catch, I totally overlooked the Fortran stuff. I wonder if it's worth having a centralized script to call different physics' Fortran version? But also we probably don't want to maintain the Fortran version beyond the porting effort.